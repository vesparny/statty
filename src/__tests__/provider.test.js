import React from 'react'
import { render } from 'enzyme'
import serializer from 'enzyme-to-json/serializer'
import { Provider, State } from '../index'

expect.addSnapshotSerializer(serializer)

test('Provider passes context down', () => {
  expect(
    render(
      <Provider state={{ state: 'ok' }}>
        <State
          children={(state, update) =>
            <span>
              {JSON.stringify(state)}
            </span>}
        />
      </Provider>
    )
  ).toMatchSnapshot()
})
