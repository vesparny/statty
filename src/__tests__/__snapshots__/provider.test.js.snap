// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Provider passes context down 1`] = `
<span>
  {"state":"ok"}
</span>
`;
